dstyle.dd

```diff
@@ -298,6 +299,13 @@ else if (…)
 -------------------------------
     $(LI Selective imports should have a space before and after the colon (`:`) like
     `import std.range : zip`)
+    $(LI Local, selective imports should be preferred over global imports)
+    $(LI In $(I non-templated) functions should be annotated with matching attributes (`@nogc`, `@safe`, `pure`, `nothrow`))
```

Comment:
In non-templated functions, WHAT should be annotated? Did you mean to not add "In"?